ci.mk

```diff
@@ -16,6 +16,14 @@ CLANG-TIDY?=clang-tidy-$(CLANG_VERSION)
 
 .SECONDEXPANSION:
 
+#############################################################################
+
+# Target requirements Each non build target, should be added as a .PHONY target
+# prerequiste. The non_build_targets variable should be used to add targets
+# that do not perform a build step.
```

Comment:
Is "Target Requirements" supposed to be a title? If so, I think it should be on a separate line and be more clear.

Also, each requirement should be on a separate point.

```suggestion
# Requirements for new targets:
# - Each non build target, should be added as a .PHONY target prerequiste.
# - The non_build_targets variable should be used to add targets that do not
#   perform a build step.
```

Reply:
Commited.-

Reply:
@josecm Kindly resolve this conversation if the changes attend to your comments. Thank you.